automatic/openssl.light/update.ps1

```diff
@@ -44,15 +44,15 @@ function global:au_GetLatest {
 
   if ([version]$version32 -lt [version]'1.1.1') {
     # Because previous package updates used dates, we need a number that ends up being larger
-    $revisionLen = '8' # Will end up with version '1.1.0.40000000'
+    $revisionLen = '6' # Will end up with version '1.1.0.40000000'
```

Comment:
I think this comment is now incorrect, right?

Reply:
Actually, it's still correct.
the helper `Get-FixVersion` works a little differently, it takes the amount of `0` to have between the actual version and the fix number.